apps/prairielearn/elements/pl-template/pl-template.py

```diff
@@ -32,7 +32,9 @@
 
 
 def check_tags(element_html: str) -> None:
-    element_list = cast(list, lxml.html.fragments_fromstring(element_html))
+    element_list = cast(
```

Comment:
Isn't this cast incorrect? The first element could be a string (see below).

Reply:
Yes, see if you can remove the potential string first.

Reply:
Fixed in [cb2a7ef](https://github.com/PrairieLearn/PrairieLearn/pull/11251/commits/cb2a7ef1058913a469a2c628bbdc1b46adc90bfc).